### ISSUE TITLE
Unable to open zip file error

### ISSUE THREAD

**Issue by aknight2015** [devnoname120/vhbb#105]
Whenever I try to download and install anything it downloads the data, upzips it, but is unable to install the vpk file.

**Comment by theAkito** [comment]
Did you try different zip files from different providers?

**Comment by aknight2015** [comment]
Yeah. I deleted the apps directory in the data folder, uninstalled it,
and reinstalled. That solved it.
